docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -39,13 +39,17 @@ Configuration
 
 To configure the Kafka connector, create a catalog properties file
 ``etc/catalog/kafka.properties`` with the following contents,
-replacing the properties as appropriate:
+replacing the properties as appropriate. In some cases, such as when using
```

Comment:
I don't like this form `In some cases`.  I think it is more about reusing existing configuration for Kafka in Trino. To me it should be separate paragraph.

@mosabua wdyt?

See for example: https://trino.io/docs/current/connector/hive.html#hdfs-configuration

Reply:
No problem. I separated `In some cases...`